src/tiled/objectselectionitem.cpp

```diff
@@ -147,6 +151,11 @@ class MapObjectOutline : public QGraphicsItem
 private:
     QRectF mBoundingRect;
     MapObject *mObject;
+
+    // Timer is used to call update function repeatedly to animate selection
+    QTimer *updateTimer = new QTimer();
+    // Offset simulates the marching ants effect
+    int offset=0;
```

Comment:
Coding style: please put spaces around the `=`.

Also, this would actually be the first time initialization is done at declaration time in Tiled's source code. But that is no problem since the Windows, macOS and Linux builds have all gone through fine.